### PR TITLE
simplify app menu / architecture

### LINKED ISSUE #128: Simplify NiAppMenu
With the recent removal of viewing lists of delegators, the receive page, and the nomination page, the sidebar seems overloaded with section titles. The page that do exist are a little empty and the content seems too thinly spread out when they should be shown together.

## Current Navigation

* **WALLET**
  * Balances
  * Send
  * Transactions
* **GOVERN**
  * Proposals
* **STAKE**
  * Delegate
* **MONITOR**
  * Blockchain
  * Validators

## Proposed Navigation
I want to propose a primary app navigation with just four buttons.

### 1. Wallet

The wallet page shows your primary address and balances on the top of the page. Below that is a list of most recent 20-30 transactions.

*Page Nav*: [Index], Transaction History (this page is a longer version of the recent transactions list)

*Page Actions*: Send Tokens (pressing "Send Tokens" will create a modal popup allowing you to fill in details about the tx).

### 2. Govern

The govern page shows a list of current proposals.

*Page Nav*: [Index], Finished Proposals (finished proposals are proposals that have already either been accepted or rejected)

### 3. Stake
The delegation game is not a thing anymore. The most important thing users will want to see on the Stake page is their current basket of validators (it's sort of like an index fund).

*Page Nav*: [Index], Edit Basket of Validators (this page shows a detailed, filterable list of all current validators)

### 4. Monitor
The monitor page shows important blockchain details at the top, and then a list of validators below that. The list of validators can be sorted and filtered.

*Page Nav*: [Index], Search Blockchain (this page will let you search for transactions, blocks, etc. in the cosmos network)

Thoughts on this?

### PR DESCRIPTION
**Pull request by jolesbi**
closes #128

### CODE REVIEW

File: app/src/renderer/components/common/NiUserPane.vue [hunk 1]
```diff
@@ -5,18 +5,7 @@ list-item.ni-li-user(
...
-    type="link"
-    icon="exit_to_app"
-    @click.native="signOut"
-    title="Sign Out")
-list-item.ni-li-user(
-  v-else
-  @click.native="openSession"
-  icon="exit_to_app"
-  title="Sign In")
+  :title="user.account")
```
Comment (faboweb) [inline app/src/renderer/components/common/NiUserPane.vue]: Users in production mode will not be able to sign out right?

Comment (jolesbi) [reply app/src/renderer/components/common/NiUserPane.vue]: users should be able to sign out from their profile page in the top bar. this is left over from when you could use the app before signing in. IMO this area should never change. should just be user's name.

Comment (faboweb) [reply app/src/renderer/components/common/NiUserPane.vue]: User profile is not production ready I think. (May be wrong) This was the original idea behind leaving the sign-out in the menu for now.

Comment (jolesbi) [reply app/src/renderer/components/common/NiUserPane.vue]: yep you're right. user profile is not ready for production (missing a lot of data). i created a ticket for this (#322). 

i'll bring back the sign out button.

**Comment by nylira** [comment]
This is really nice!

**Pull request merged**